ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-join.json

```diff
@@ -1,99 +1,105 @@
 {
   "tests": [
     {
-      "name": "Should fail on right non-key attribute for inner-join",
```

Comment:
This diff is confusion... The test (and the next 3 tests) got moved to `table-table-join.json`